### PR TITLE
Twittercards

### PR DESCRIPTION
**Pull request by gusreiber**

### CODE REVIEW

File: site/content/feedback/_index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
```
Comment (liamawhite) [inline site/content/feedback/_index.md]: You need to rebase or reset from master otherwise you'll undo the changes I made on this page.

Comment (gusreiber) [reply site/content/feedback/_index.md]: @liamawhite ...this file is being added to Master in this PR. It isn't in Master prior or in any other PR. Thus, you, being last in, will need to apply your changes to this file. Rebasing isn't going to do anything because the file doesn't exist in any base prior.

**Comment by liamawhite** [comment]
closing in favour of #98
